src/plugins/regions.ts

```diff
@@ -73,8 +73,10 @@ class SingleRegion extends EventEmitter<RegionEvents> {
 
   constructor(params: RegionParams, private totalDuration: number) {
     super()
-
     this.id = params.id || `region-${Math.random().toString(32).slice(2)}`
+    if (Array.isArray(this.id)) {
+      this.id = this.id.join(' ')
+    }
```

Comment:
Subsequently, here I would assign it only as a string. I.e.
```suggestion
    if (typeof params.id === 'string) {
      this.id = params.id
    } else if (Array.isArray(params.id)) {
      this.id = params.id.join(' ')
    } else {
      this.id = `region-${Math.random().toString(32).slice(2)}`
    }
```